File: Listas de Exercícios (Python)/Python/Lista 3 - Sala/11.py
print("5 Números inteiros - Menor e maior valor\n")

print("Digite-os: ")
N1=int(input("1º: "))
N2=int(input("2º: "))
N3=int(input("3º: "))
N4=int(input("4º: "))
N5=int(input("5º: "))

if (N1>N2 and N1>N3 and N1>N4 and N1>N5):
    print("Maior número: ", N1)
elif (N2>N1 and N2>N3 and N2>N4 and N2>N5):
    print("Maior número: ", N2)
elif (N3>N1 and N3>N2 and N3>N4 and N3>N5):
    print("Maior número: ", N3)
elif (N4>N1 and N4>N2 and N4>N3 and N4>N5):
    print("Maior número: ", N4)
elif (N5>N1 and N5>N2 and N5>N3 and N5>N4):
    print("Maior número: ", N5)
elif (N1==N2 or N1==N3 or N1==N4 or N1==N5 or N2==N3 or N2==N4 or N2==N5 or N3==N4 or N3==N5 or N4==N5):
    print("\nHá valores iguais! Eles são os maiores números!")

if (N1<N2 and N1<N3 and N1<N4 and N1<N5):
    print("Menor número: ", N1)
elif (N2<N1 and N2<N3 and N2<N4 and N2<N5):
    print("Menor número: ", N2)
elif (N3<N1 and N3<N2 and N3<N4 and N3<N5):
    print("Menor número: ", N3)
elif (N4<N1 and N4<N2 and N4<N3 and N4<N5):
    print("Menor número: ", N4)
elif (N5>N1 and N5>N2 and N5>N3 and N5>N4):
    print("Maior número: ", N5)
elif (N1==N2 or N1==N3 or N1==N4 or N1==N5 or N2==N3 or N2==N4 or N2==N5 or N3==N4 or N3==N5 or N4==N5):
    print("\nHá valores iguais! Eles são os menores números!")

File: 1th grade/Listas de Exercícios (Back-end [SPOLOGP])/C/Lista 3 - Sala/11.c
#include <stdio.h>

int main()
{
    int N1, N2, N3, N4, N5;
    printf("5 Números inteiros - Menor e maior valor\n\n");
    
    printf("Digite-os:\n");
    printf("1º: "); scanf("%d", &N1);
    printf("2º: "); scanf("%d", &N2);
    printf("3º: "); scanf("%d", &N3);
    printf("4º: "); scanf("%d", &N4);
    printf("5º: "); scanf("%d", &N5);
    
    if (N1>N2 && N1>N3 && N1>N4 && N1>N5){
        printf("Maior número: %d\n", N1);
    }else if (N2>N1 && N2>N3 && N2>N4 && N2>N5){
        printf("Maior número: %d\n", N2);
    }else if (N3>N1 && N3>N2 && N3>N4 && N3>N5){
        printf("Maior número: %d\n", N3);
    }else if (N4>N1 && N4>N2 && N4>N3 && N4>N5){
        printf("Maior número: %d\n", N4);
    }else if (N5>N1 && N5>N2 && N5>N3 && N5>N4){
        printf("Maior número: %d\n", N5);
    }else if (N1==N2 || N1==N3 || N1==N4 || N1==N5 || N2==N3 || N2==N4 || N2==N5 || N3==N4 || N3==N5 || N4==N5){
        printf("\nHá valores iguais! Eles são os maiores números!\n");
    }
    
    if (N1<N2 && N1<N3 && N1<N4 && N1<N5){
        printf("Menor número: %d\n", N1);
    }else if (N2<N1 && N2<N3 && N2<N4 && N2<N5){
        printf("Menor número: %d\n", N2);
    }else if (N3<N1 && N3<N2 && N3<N4 && N3<N5){
        printf("Menor número: %d\n", N3);
    }else if (N4<N1 && N4<N2 && N4<N3 && N4<N5){
        printf("Menor número: %d\n", N4);
    }else if (N5<N1 && N5<N2 && N5<N3 && N5<N4){
        printf("Menor número: %d\n", N5);
    }else if (N1==N2 || N1==N3 || N1==N4 || N1==N5 || N2==N3 || N2==N4 || N2==N5 || N3==N4 || N3==N5 || N4==N5){
        printf("\nHá valores iguais!\nEles são os maiores!\n");
    }
}